### ISSUE TITLE
Freezing Player

### ISSUE THREAD

**Issue by StellarExile** [KOTORCommunityPatches/K1_Community_Patch#403]
During the temple scene, if the player chooses the dark side path, it's possible for the player to freeze after Jolee says "Don't do this kid, I don't want to, but I'll fight you if I have to..."

**Comment by DarthParametric** [comment]
Hrm, I don't recall seeing this before. You're not using other mods?

**Comment by StellarExile** [comment]
Yeah, although I think it doesn't happen always, I believe it may be script or camera related. I know it happens in vanilla and with the patch.
